compiler/rustc_ast/src/ast.rs

```diff
@@ -2993,7 +2993,7 @@ pub enum ItemKind {
     Union(VariantData, Generics),
     /// A trait declaration (`trait`).
     ///
-    /// E.g., `trait Foo { .. }`, `trait Foo<T> { .. }` or `auto trait Foo {}`.
```

Comment:
Not sure whether it's worth having an example for deleted syntax basically. I think obvious that an auto trait is represented as a trait and adding this example is just confusing.

Reply:
Absolutely, I've actually already removed in a later commit (*Update the compiler itself …*, 53e59fa0a63afdde2e42d4021c4634eaebfdbc41). Sorry that's not really obvious.